include/oneapi/dpl/pstl/ranges/nanorange.hpp

```diff
@@ -527,8 +527,7 @@ namespace detail
 {
 
 template <typename T, typename U>
-constexpr bool same_decayed_v =
-    ::std::is_same<T, ::std::decay_t<T>>::value&& ::std::is_same<U, ::std::decay_t<U>>::value;
+constexpr bool same_decayed_v = ::std::is_same_v<T, ::std::decay_t<T>> && ::std::is_same_v<U, ::std::decay_t<U>>;
```

Comment:
This and the other change in this file are probably better suited for the #1173 PR.

Reply:
Good point. Moved into #1173 PR.